auditbeat/module/auditd/ecs_categorization.yml

```diff
@@ -0,0 +1,320 @@
+    # AUDIT_USER_CHAUTHTOK - User acct password or pin changed // should this be classified EventTypeUserAccount?
+    # AUDIT_USER_ERR - User acct state error
+    # AUDIT_CRED_REFR - User credential refreshed
+    # AUDIT_USER_LOGIN - User has logged in
+    # AUDIT_USER_LOGOUT - User has logged out
+    # AUDIT_GRP_AUTH - Authentication for group password
+    categories:
+      - authentication
+    default_types:
+      - info
+    types:
+      AUDIT_USER_LOGIN:
+        - start
+      AUDIT_USER_LOGOUT:
+        - end
+
+  EventTypeVirt:
+    # AUDIT_VIRT_CONTROL - Start, Pause, Stop VM
+    # AUDIT_VIRT_RESOURCE - Resource assignment
+    # AUDIT_VIRT_MACHINE_ID - Binding of label to VM
+    # AUDIT_VIRT_INTEGRITY_CHECK - Guest integrity results
+    # AUDIT_VIRT_CREATE - Creation of guest image
+    # AUDIT_VIRT_DESTROY - Destruction of guest image
+    # AUDIT_VIRT_MIGRATE_IN - Inbound guest migration info
+    # AUDIT_VIRT_MIGRATE_OUT - Outbound guest migration info
+    categories:
+      - host
+    default_types:
+      - info
+
+  EventTypeUserspace:
+    # AUDIT_CHGRP_ID - User space group ID changed
+    # AUDIT_TEST - Used for test success messages
+    # AUDIT_TRUSTED_APP - Trusted app msg - freestyle text
+    # AUDIT_USER_CMD - User shell command and args
+    # AUDIT_CHUSER_ID - Changed user ID supplemental data
+    categories:
+      - process
+    default_types:
+      - info
+    types:
+      AUDIT_CHUSER_ID:
+        - change
+      AUDIT_CHGRP_ID:
+        - change
+
+  EventTypeSystemServices:
+    # AUDIT_KERNEL - Asynchronous audit record. NOT A REQUEST.
+    # AUDIT_SYSTEM_BOOT - System boot
+    # AUDIT_SYSTEM_SHUTDOWN - System shutdown
+    # AUDIT_SYSTEM_RUNLEVEL - System runlevel change
+    # AUDIT_SERVICE_START - Service (daemon) start
+    # AUDIT_SERVICE_STOP - Service (daemon) stop
+    categories:
+      - host
+    default_types:
+      - info
+    types:
+      AUDIT_SYSTEM_BOOT:
+        - start
+      AUDIT_SYSTEM_RUNLEVEL:
+        - change
+      AUDIT_SYSTEM_SHUTDOWN:
+        - end
+
+  EventTypeAuditDaemon:
+    # AUDIT_DAEMON_START - Daemon startup record
+    # AUDIT_DAEMON_END - Daemon normal stop record
+    # AUDIT_DAEMON_ABORT - Daemon error stop record
+    # AUDIT_DAEMON_CONFIG - Daemon config change
+    categories:
+      - process
+    default_types:
+      - info
+    types:
+      AUDIT_DAEMON_START:
+        - start
+      AUDIT_DAEMON_END:
+        - end
+
+  EventTypeDACDecision:
+    # AUDIT_SECCOMP - Secure Computing event
+    categories:
+      - process
+    default_types:
+      - change
+
+  EventTypeConfig:
+    # AUDIT_USYS_CONFIG - User space system config change
+    # AUDIT_CONFIG_CHANGE - Audit system configuration change
+    # AUDIT_NETFILTER_CFG - Netfilter chain modifications
+    # AUDIT_FEATURE_CHANGE - audit log listing feature changes
+    # AUDIT_REPLACE - Replace auditd if this packet unanswerd
+    categories:
+    #   - config
```

Comment:
It's fine not to assign a category for now, until we define a more appropriate one 👍